src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -162,11 +162,12 @@ class SoftmaxRegression
    * @param optimizer Desired optimizer.
    * @return Objective value of the final point.
    */
-  template<typename OptimizerType = ens::L_BFGS>
+  template<typename OptimizerType = ens::L_BFGS,typename... CallbackTypes>
```

Comment:
Would be great if we could comment on the `CallbackTypes`, see https://github.com/mlpack/mlpack/pull/2076 for more details.

Reply:
ok

Reply:
```suggestion
  template<typename OptimizerType = ens::L_BFGS, typename... CallbackTypes>
```
:)